components/new-contribution-flow/index.js

```diff
@@ -282,6 +294,25 @@ class ContributionFlow extends React.Component {
       }
     }
 
+    // Check that the contributor is not blocked from contributing to the collective
+    const collectiveBlockedCategories = get(
+      this.props.collective,
+      'settings.moderation.rejectedContributionCategories',
+      [],
+    );
+    if (collectiveBlockedCategories) {
+      const { data } = await this.props.client.query({
+        query: accountCategoriesQuery,
+        variables: { collectiveSlug: stepProfile.slug },
+        context: API_V2_CONTEXT,
+      });
+      const contributorBlockedCategories = get(data, 'account.categories', []);
```

Comment:
These are not "Blocked Categories". I'm not against something explicit but if we don't have better, I would stick to "contributorCategories".